api/identifiers.proto

```diff
@@ -108,3 +108,13 @@ message EndDeviceVersionIdentifiers {
   string firmware_version = 4 [(validate.rules).string.max_len = 32];
   string band_id = 5 [(validate.rules).string.max_len = 32, (gogoproto.customname) = "BandID"];
 }
+
+// Identifies a Network Server.
+message NetworkIdentifiers {
+  // LoRa Alliance NetID.
+  string net_id = 1;
```

Comment:
This is TTS API, so this should be `bytes` and with the `gogoproto.customtype` tag. I know we're stepping away from gogoproto, but we need to stay consistent with both protobuf and the JSON API. Due to the compatibility commitment, we will make sure that JSON NetID will stay represented in hex.